code/core/src/components/components/Button/Button.tsx

```diff
@@ -142,6 +142,7 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
             size={size}
             padding={padding}
             disabled={disabled}
+            readOnly={readOnly}
```

Comment:
This may be duplicated because I could swear I've written a comment already but GitHub is not showing me the comment. Feel free to discard accordingly.

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/readonly not applicable to buttons

Instead I would try either this first snippet + properly block event handlers if either disabled or readOnly is on.

```suggestion
            aria-disabled={disabled || readOnly}
            role={readOnly ? 'presentation' : undefined}
```

Or this one if you don't wanna handle the side effects of fixing up the use of `disabled`.

```suggestion
            disabled={disabled || readOnly}
            role={readOnly ? 'presentation' : undefined}
```

In any case, depending on what readOnly is supposed to do, we may still want it to be keyboard focusable / visible in the a11y object model, but without a `button` role.

Reply:
I'm ok with `readOnly`, I just wanted to make sure I'd given you a WCAG markup to work with :)

Off the top of my head, I'd use a name like `forDemos` or `forShowcase`, something that's rooted in the intended use case rather than the behaviour, so that people don't accidentally use this behaviour for other purposes (like actually disabling the button).